ui/src/clients.ts

```diff
@@ -0,0 +1,22 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ResourceManagementClient } from '@azure/arm-resources';
+import { StorageManagementClient } from '@azure/arm-storage';
+import { SubscriptionClient } from '@azure/arm-subscriptions';
+import * as types from '../index';
+import { createAzureClient, createAzureSubscriptionClient } from './createAzureClient';
+

```

Comment:
How about a comment to explain what's going on - not sure this is obvious. Something like:

> // Lazy-load `@azure` packages to improve startup performance. 
> // NOTE: The client is the only import that matters, the rest of the types disappear when compiled to JavaScript

Reply:
Good idea!